solutions/azure-hub-spoke-connected-group/bicep/modules/avnm.bicep

```diff
@@ -0,0 +1,315 @@
+param location string
+param prodNetworkGroupMembers array
+param nonProdNetworkGroupMembers array
+param hubVnetId string
+param deployVpnGateway bool
+param deployDefaultDenySecurityAdminRules bool
+
+@description('This is the Azure Virtual Network Manager which will be used to implement the connected group for spoke-to-spoke connectivity.')
+resource networkManager 'Microsoft.Network/networkManagers@2022-05-01' = {
+  name: 'avnm-${location}'
```

Comment:
I'm glad to see this being called out as a regional resource. As hub-spokes themselves are regional, it makes sense that this is a 1 manager-to-many-hubs, but all within the same region.  Good guidance!